sematic/ui/src/components/PipelineBar.tsx

```diff
@@ -68,16 +77,36 @@ export default function PipelineBar(props: {
     []
   );
 
+  const fetchResolution = useCallback(
+    (run: Run | undefined, onResults: (resolution: Resolution | undefined) => void) => {
```

Comment:
It seems `onResults` is always set to `setResolution` in the 3 calls below.
Why not hard-code it on line 87 and 91 below, and add `setResolution` to the dependency array on line 96?

Reply:
Done.